pkgs/by-name/fl/flashgbx/package.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, copyDesktopItems
+, makeDesktopItem
+, qt6
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "flashgbx";
+  version = "3.36";
+
+  src = fetchFromGitHub {
+    repo = "FlashGBX";
+    owner = "lesserkuma";
+    rev = "${version}";
+    sha256 = "sha256-MbeQKdmAGt4bTSleU1Dlve7fLDfldS2gIYUCIbgDAk4=";
+  };
+
+  desktopItems = let
+    common = {
+      name = "flashgbx";
+      desktopName = "FlashGBX UI";
+      icon = "flashgbx";
+      exec = "flashgbx";
+      comment = "UI for reading and writing Game Boy and Game Boy Advance cartridges";
+      categories = [ "Utility" ];
+    };
+  in builtins.map makeDesktopItem [
+    common
+    (common // {
+      name = "${common.name}";
+      exec = "${common.exec}";
+    })
```

Comment:
it's unclear to me what exactly this is doing.  as far as i can tell it is overriding `common.name` with `common.name`, which seems like it wouldn't do anything?

Reply:
Thanks for pointing this out. This was a remnant from the example I started from. I have now looked at a few other examples of `makeDesktopItem` and adapted the code, removing the mapping.